src/ProjectOrigin.Vault/Database/Postgres/Scripts/v2/v2-0010.sql

```diff
@@ -0,0 +1,12 @@
+CREATE PROCEDURE expire_slices(older_than_date timestamp with time zone, expire_state_int integer, available_state_int integer)
+AS $$
+BEGIN
+    UPDATE wallet_slices ws
+    SET state = expire_state_int
```

Comment:
I don't understand the first question?

I have those 2 because then I can get the parameter from the C# code instead of hardcoding it here. I was a bit torn about this